src/data-source/DataSource.ts

```diff
@@ -407,10 +407,25 @@ export class DataSource {
     /**
      * Reverts last executed migration.
      * Can be used only after connection to the database is established.
+     *
+     * @deprecated use `revertMigration` instead
      */
     async undoLastMigration(options?: {
         transaction?: "all" | "none" | "each"
         fake?: boolean
+        until?: string
+    }): Promise<void> {
+        await this.revertMigration(options)
+    }
+
+    /**
+     * Reverts last executed migration.
+     * Can be used only after connection to the database is established.
+     */
+    async revertMigration(options?: {
```

Comment:
If I can make a few suggestions:

- this method should be named `revertMigrations` (plural) for clarity
- the `until` parameter should be a required argument
- the `until` option should be inclusive rather than exclusive. That is, the named migration should be reverted as well as those after it. This will be easier to use?
- the `undoLastMigration` method should not take an `until` option. It should look up the name of the last migration and pass it to `revertMigrations`